client/web/src/enterprise/codeintel/list/CodeIntelUploadsPage.tsx

```diff
@@ -103,56 +104,64 @@ export const CodeIntelUploadsPage: FunctionComponent<CodeIntelUploadsPageProps>
-                            still provide code intelligence for historic and branch commits.
-                        </p>
+                        Current uploads provide code intelligence for the latest commit on the default branch and are
+                        used in cross-repository <em>Find References</em> requests. Non-current uploads may still
+                        provide code intelligence for historic and branch commits.
                     </>
                 }
+                className="mb-3"
             />
 
-            {repo && commitGraphMetadata && (
-                <CommitGraphMetadata
-                    stale={commitGraphMetadata.stale}
-                    updatedAt={commitGraphMetadata.updatedAt}
-                    now={now}
-                />
-            )}
+            <Container>
+                {repo && commitGraphMetadata && (
+                    <CommitGraphMetadata
+                        stale={commitGraphMetadata.stale}
+                        updatedAt={commitGraphMetadata.updatedAt}
+                        now={now}
+                    />
+                )}
 
-            <div className="list-group position-relative">
-                <FilteredConnection<LsifUploadFields, Omit<CodeIntelUploadNodeProps, 'node'>>
-                    listComponent="div"
-                    listClassName="codeintel-uploads__grid mb-3"
-                    noun="upload"
-                    pluralNoun="uploads"
-                    nodeComponent={CodeIntelUploadNode}
-                    nodeComponentProps={{ now }}
-                    queryConnection={queryUploads}
-                    history={props.history}
-                    location={props.location}
-                    cursorPaging={true}
-                    filters={filters}
-                    defaultFilter="current"
-                />
-            </div>
+                <div className="list-group position-relative">
+                    <FilteredConnection<LsifUploadFields, Omit<CodeIntelUploadNodeProps, 'node'>>
+                        listComponent="div"
+                        listClassName="codeintel-uploads__grid"
+                        noun="upload"
+                        pluralNoun="uploads"
+                        nodeComponent={CodeIntelUploadNode}
+                        nodeComponentProps={{ now }}
+                        queryConnection={queryUploads}
+                        history={props.history}
+                        location={props.location}
+                        cursorPaging={true}
+                        filters={filters}
+                        defaultFilter="current"
+                        emptyElement={<EmptyLSIFUploadsElement />}
+                    />
+                </div>
+            </Container>
         </div>
     )
 }
+
+const EmptyLSIFUploadsElement: React.FunctionComponent<{}> = () => (
```

Comment:
nit: empty object can be omitted:

```suggestion
const EmptyLSIFUploadsElement: React.FunctionComponent = () => (
```

Reply:
I thought empty meant `any`, but apparently that's not the case. Sweet